### ISSUE TITLE
Workflow "Create a  new app" fails

### ISSUE THREAD

**Issue by achim-t** [microsoft/AL-Go#98]
https://github.com/achim-t/shiny-octo-computing-machine/actions/runs/2263871095

I was following scenario 1.

```
git push ***github.com/achim-t/shiny-octo-computing-machine
remote: Permission to achim-t/shiny-octo-computing-machine.git denied to github-actions[bot].
fatal: unable to access 'https://github.com/achim-t/shiny-octo-computing-machine/': The requested URL returned error: 403
Error: Adding a new app failed.
```

**Comment by freddydk** [comment]
This issue is caused by a change in GitHub defaults.
It is fixed in the preview of AL-Go - but for now, you can go to:
https://github.com/achim-t/shiny-octo-computing-machine/settings/actions
and set default workflow permissions to read and write:
![image](https://user-images.githubusercontent.com/10775043/166462079-479eb1a7-b276-47d2-bc20-8f719de030cb.png)

In the next version, permissions are specified in the workflows and this setting is no longer needed.

**Comment by achim-t** [comment]
Yes, that worked.

**Comment by freddydk** [comment]
A fix for this is shipped in v1.5
